articles/how-to-guides/create-an-application-template/create-an-application-template.md

```diff
@@ -2,3 +2,83 @@
 uid: how-to-guides.create-an-application-template
 ---
 # Create an Application Template
+
+Do you find yourself creating new applications in Intent Architect by choosing a very simple application template and customizing it by installing lots of modules?
+Working in a small team or organization can become daunting when someone else needs to create a new application and are not sure what the configuration settings should be.
+
+In this how-to we will explain how to setup your own team or organizational application template list which can run side-by-side to the official application template list and allows developers not only to create their own applications with ease, but they can easily customize it without having too much in-depth knowledge upon creation.
```

Comment:
In this how-to we will explain how to setup your own team or organizational application template list which can run side-by-side to the official application template list and allows developers not only to create their own applications with ease, but they can easily customize it without ~having~ requiring too much in-depth knowledge upon creation.